rectangle.js

```diff
@@ -0,0 +1,29 @@
+function Rectangle(width, height) {
+  this.width = width
+  this.height = height
+}
+
+Rectangle.prototype = {
```

Comment:
we discussed the differences of using the class es6 syntax vs this syntax in class today. please let me know if you have questions 👍